src/olympia/editors/templates/editors/emails/pending_to_public.ltxt

```diff
@@ -1,5 +1,5 @@
 {% extends "editors/emails/base.ltxt" %}{% block content %}
-Your {{ addon_type }}, {{ name }} {{ number }}, has been fully reviewed and is now available for download in our gallery at {{ addon_url }}
+Your add-on, {{ name }} {{ number }}, has been fully reviewed and is now available for download in our gallery at {{ addon_url }}
```

Comment:
Seems weird we're not using the `ADDON_TYPE` constants to say 'Extension', 'Dictionary', etc, but if we always say add-on now, then meh.